src/store/modules/inbox.js

```diff
@@ -2,34 +2,38 @@
 // import InboxService from '../../services/inbox/inbox';
 
 const state = {
-  selectedInboxItemId: 5,
+  selectedInboxItemId: 0,
   mobileDrawerOpen: false,
 };
 
 // getters
 const getters = {
-  getInboxItems(state) {
-    let inboxItems = [
-      {
-        id: 1,
-        senderIcon: "VC",
-        senderName: "Virtual Concierge",
-        teaserText: "Hi, how can I help you?",
-        dayRead: "",
+  getInboxItems(state, getters, rootState, rootGetters) {
+    let inboxItems = [];
+
+    rootGetters["chatMessages/getAllChatMessages"].forEach((item) => {
+      inboxItems.push({
+        id: item.id,
+        senderIcon: item.senderIcon,
+        senderName: item.senderName,
+        teaserText: "",
+        dayRead: "tues",
         selected: false,
         type: "chat",
-      },
-      {
-        id: 2,
-        senderIcon: "Mail",
-        senderName: "Job Bank",
-        teaserText:
-          "You are receiving this email because you recently completed an application for Employment Insurance.",
-        dayRead: "Weds",
+      });
+    });
+
+    rootGetters["emails/getMailObject"].forEach((item) => {
+      inboxItems.push({
+        id: item.id,
+        senderIcon: item.senderIcon,
+        senderName: item.senderName,
+        teaserText: "",
+        dayRead: "wed",
```

Comment:
If we can, we could call a function, pass the item, and return the date we need. Just an idea. (This can be implemented in a different PR)

P.S.: The same applies for teaserText.

Reply:
Yes, I didn't want to jump into anything complex before establishing a baseline. Once we get this PR merged that can be done

Reply:
ok sounds good to me.